src/components/Reservations/MyReservation.module.css

```diff
@@ -0,0 +1,38 @@
+.table {
+  border: 1px solid black;
+  margin: 0 auto;
+  width: 100%;
+  text-align: center;
+}
+
+.table th {
+  border: 1px solid black;
+  padding: 10px;
+  background-color: #97bf0f;
```

Comment:
Yes this border, you can go ahead and remove it, this will make the page look cleaner